whist/server/cli.py

```diff
@@ -1,8 +1,12 @@
-"""Main entrypoint"""
-
+"""CLI entrypoint"""
 import uvicorn
 
 from whist.server import app
 
-if __name__ == '__main__':
+
+def main():
     uvicorn.run(app, host='0.0.0.0', port=8000)
```

Comment:
Intend missing?

Reply:
Nothing is missing here. the `if __name__ == '__main__'` got moved

Reply:
In the diff it looks like the body of `main()` has no indent

Reply:
nope

Reply:
Weird, it was only on the mobile ...